include/deferrable.h

```diff
@@ -0,0 +1,73 @@
+#ifndef __DEFERRABLE_H__
+#define __DEFERRABLE_H__
+
+/*
+* File      : deferrable.h
+* This file is part of RT-Thread RTOS
+* COPYRIGHT (C) 2006, RT-Thread Development Team
+*
+* The license and distribution terms for this file may be
+* found in the file LICENSE in this distribution or at
+* http://www.rt-thread.org/license/LICENSE.
+*
+* Change Logs:
+* Date           Author                         Notes
+* 2021-12-08    Nicola di Gruttola Giardino     First version
+*
+*/
+
+#include <rtthread.h>
+
+#define DEBUG_DEFERRABLE 0
+
+#define DEFERRABLE_SERVER_STACK_SIZE 4096
+
+rt_uint8_t rt_deferrable_stack[DEFERRABLE_SERVER_STACK_SIZE];
+
```

Comment:
Sorry for the lack of comments in the .h file, if you need any clarifications just ask..

Reply:
Don't worry. You don't need to fix that. leave that to me. I will modify them. I am on it.

Reply:
RT-Thread has a [workqueue](https://github.com/RT-Thread/rt-thread/blob/master/components/drivers/src/workqueue.c). Please consider what different between the workqueue and yours.